### PR TITLE
[PL-38229] Add auto scale using replicas topic - delegate

### PR DESCRIPTION
**Pull request by brian-f-harness**
Add auto scale using replicas topic - delegate

For reviewers, [a preview](https://645a9674af852322bf1793ad--harness-developer.netlify.app/docs/platform/Delegates/manage-delegates/auto-scale-using-replicas) is available.

## What Type of PR is This?

- [ ] Issue
- [x] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [x] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

**Review by pratmit — COMMENTED** [review]
Need to specify that this is for the helm chart deployment. Can you add the helm values.yaml link.

**Comment by brian-f-harness** [comment]
@pratmit I added a commit with the link to the Helm chart and updated the preview.

**Review by pratmit — COMMENTED** [review]
Can you please add a note that this can be added to the harness delegate yaml file as well in addition to helm chart?

**Review by pratmit — APPROVED** [review]

File: docs/platform/2_Delegates/manage-delegates/auto-scale-using-replicas.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+---
+title: Auto scale using replicas
```
Comment (douglas-j-bothwell) [inline docs/platform/2_Delegates/manage-delegates/auto-scale-using-replicas.md:2]: Is there a reason you're using "Auto scale" (two words) here and "autoscale" (one word) everywhere else?

**Review by douglas-j-bothwell — APPROVED** [review]

File: docs/platform/2_Delegates/manage-delegates/auto-scale-using-replicas.md (hunk 1 above)
Comment (brian-f-harness) [reply docs/platform/2_Delegates/manage-delegates/auto-scale-using-replicas.md:2]: Nice catch. Updating.

**Pull request merged**